cmd/pd-tso-bench/main.go

```diff
@@ -82,6 +82,9 @@ func main() {
 func showStats(ctx context.Context, durCh chan time.Duration) {
 	defer wg.Done()
 
+	statCtx, cancel := context.WithCancel(ctx)
+	defer cancel()
```

Comment:
Do we really need to call `cancel()`?
When `cancel` is called, no one is blocking on it.